impl/src/main/java/org/jboss/weld/contexts/CreationalContextImpl.java

```diff
@@ -192,6 +228,14 @@ protected Object writeReplace() throws ObjectStreamException {
         return this;
     }
 
+    // Serialization
+    protected Object readResolve() throws ObjectStreamException {
```

Comment:
Do we actually need this method?
I am struggling to see when are we going to use it.

Reply:
Removed